roles/openstack/neutron/tasks/setup.yml

```diff
@@ -52,3 +52,25 @@
   register: routers
   with_dict: "{{ provisioner.neutron.routers }}"
   when: provisioner.neutron.routers is defined
+
+- name: Create security groups
+  os_security_group:
+      cloud: "{{ provisioner.cloud | default(omit) }}"
+      name: "{{ prefix }}{{ item.value.name | default(security_group_name) }}"
+      state: present
+  register: security_groups
+  with_dict: "{{ provisioner.neutron.security_groups }}"
+  when: provisioner.neutron.security_groups is defined
+
```

Comment:
Where do we have the secgroup dict? Is it missing from openstack.yaml file?
Note - I don't think we need to expose this in spec ATM